subscriptions/subscriptions-impl/src/main/java/com/duckduckgo/subscriptions/impl/SubscriptionsManager.kt

```diff
@@ -277,10 +327,16 @@ class RealSubscriptionsManager @Inject constructor(
     }
 
     override suspend fun signOut() {
+        authRepository.getAccessTokenV2()?.run {
+            coroutineScope.launch { authClient.tryLogout(accessTokenV2 = jwt) }
```

Comment:
There is no need to specify io dispatcher anywhere in this method since there are no thread-blocking operations here. Calling `/logout` endpoint is launched in a separate coroutine just because this is a side-effect that the caller of `signOut()` shouldn't care for and wait for. This matters e.g., when the user taps "Remove subscription from device" in settings and we want the repository transition to the signed out state ASAP, so that the UI can be updated accordingly.